src/common/asb/Asb.c

```diff
@@ -2041,10 +1492,12 @@ static char* AuditEnsureZeroconfNetworkingIsDisabled(void* log)
 
 static char* AuditEnsurePermissionsOnBootloaderConfig(void* log)
 {
+    const char* value = g_desiredEnsurePermissionsOnBootloaderConfig ? 
+        g_desiredEnsurePermissionsOnBootloaderConfig : g_defaultEnsurePermissionsOnBootloaderConfig;
     char* reason = NULL;
-    CheckFileAccess("/boot/grub/grub.cfg", 0, 0, 400, &reason, log);
-    CheckFileAccess("/boot/grub/grub.conf", 0, 0, 400, &reason, log);
-    CheckFileAccess("/boot/grub2/grub.cfg", 0, 0, 400, &reason, log);
+    CheckFileAccess("/boot/grub/grub.cfg", 0, 0, (unsigned int)atoi(value), &reason, log);
```

Comment:
nit: Why are we doing the cast to `unassigned int` here?

Reply:
Because of the CheckFileAccess definition. This gives me an idea to improve this a bit. Stay tuned.